src/main/java/io/github/townyadvanced/townycombat/listeners/TownyCombatBukkitEventListener.java

```diff
@@ -112,6 +112,18 @@ public void on (PlayerJoinEvent event) {
 		TownyCombatMovementUtil.adjustPlayerAndMountSpeeds(event.getPlayer());
 	}
 
+	@EventHandler (ignoreCancelled = true)
+	public void on (EntityDamageEvent event) {
+		if (event.getEntity() instanceof AbstractHorse) {
```

Comment:
It might make sense to make sure the horse has a rider, otherwise this will mean every horse on the server is invulnerable to fire all the time?

Reply:
Done